.gitignore

```diff
@@ -30,6 +30,7 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+*.pyc
```

Comment:
Since you are excluding the `*.pyc` files, would you mind including the deletion of https://github.com/rcbops/molecule-validate-nova-deploy/blob/master/molecule/default/tests/test_default.pyc in this PR?